templates/nodegroupconfiguration-drbd-install-altlinux-like.yaml

```diff
@@ -85,7 +86,9 @@ spec:
         rmmod drbd || true
       fi
 
-      if [ "${current_version}" == "${desired_version}" ]; then
+      if { echo "${current_version}"; echo "${force_rebuild_version}"; } | sort --version-sort --check; then
```

Comment:
We have to use semver comparsion, it is dangerous to compare strings.

9.1.2 will be higher than 9.1.14.